include/boost/math/distributions/dirichlet.hpp

```diff
@@ -50,6 +50,12 @@ inline bool check_alpha(const char *function,
                         const Policy &pol)
 {
   using RealType = typename RandomAccessContainer::value_type;
+  using std::invalid_argument;
+  if (alpha.size() < 1)
+  {
+    throw invalid_argument("Size of 'concentration parameters' must be greater than 0.");
+    return false;
```

Comment:
Do we need both a throw and a `return false`? I'd just do `throw std::invalid_argument`.

Reply:
Oh yes.